README.md

```diff
@@ -33,6 +33,12 @@ add_filter( 'graphql_jwt_auth_secret_key', function() {
 });
 ```
 
+If you are using Apache to run Wordpress you should add this to your .htaccess in order to enable Authorization header in your project
```

Comment:
@paladdins Let's update the language a bit to explain that the WordPress environment needs to receive the Authorization header for authenticated requests. Maybe something like: 

## HTTP_AUTHORIZATION
In order to use this plugin, your WordPress environment must support the HTTP_AUTHORIZATION header. In some cases, this header is not passed to WordPress because of some server configurations. 

Depending on your particular environment, you may have to research how to enable these headers, but in Apache, you can do the following in your `.htaccess`:

```
SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
```

For NGINX, this may work: https://serverfault.com/questions/511206/nginx-forward-http-auth-user#answer-511612